src/components/chatsDetailPage/Header.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
+import { BackArrowIcon, FindIcon, CallIcon, IndexIcon, InfoIcon } from '../../assets';
+import { flexCenter } from '../../styles/GlobalStyle';
+import userData from '../../assets/data/userData.json';
+import { useNavigate } from 'react-router-dom';
+import date from '../../utils/date';
+import formatDate from '../../utils/formatDate';
+
+export default function Header() {
+  const currentId = parseInt(localStorage.getItem('userId') || '');
+  const navigate = useNavigate();
+
+  function moveToChats() {
+    navigate(`/chats`);
+  }
+
+  return (
+    <>
+      <Nav>
+        <Time>{formatDate(date()).split(' ').slice(1)}</Time>
```

Comment:
이 코드도 함수로 빼서 사용하면 좋을 것 같아요,

```suggestion
function getFormattedTime() {
  return formatDate(date()).split(' ')[1]; // 예를 들어 시간만 필요하다면 인덱스 조정
}

<Time>{getFormattedTime()}</Time>
```